eva_sub_cli/orchestrator.py

```diff
@@ -111,20 +112,24 @@ def get_project_and_vcf_fasta_mapping_from_metadata_xlsx(metadata_xlsx, mapping_
             file_name = row[files_headers['File Name']]
             analysis_alias = row[files_headers['Analysis Alias']]
             reference_fasta = analysis_alias_dict[analysis_alias]
+            if not (file_name and os.path.isfile(file_name)):
+                raise FileNotFoundError(f'The variant file {file_name} provided in spreadsheet {metadata_xlsx} does not exist')
+            if not (reference_fasta and os.path.isfile(reference_fasta)):
+                raise FileNotFoundError(f'The reference fasta {reference_fasta} in spreadsheet {metadata_xlsx} does not exist')
             vcf_fasta_report_mapping.append([os.path.abspath(file_name), os.path.abspath(reference_fasta), ''])
 
     return project_title, vcf_fasta_report_mapping
 
 
-def check_validation_required(tasks, sub_config):
+def check_validation_required(tasks, sub_config, username=None, password=None):
     # Validation is mandatory so if submit is requested then VALIDATE must have run before or be requested as well
     if SUBMIT in tasks:
         if not sub_config.get(READY_FOR_SUBMISSION_TO_EVA, False):
             return True
         submission_id = sub_config.get(SUB_CLI_CONFIG_KEY_SUBMISSION_ID, None)
         if submission_id:
             try:
-                submission_status = SubmissionWSClient().get_submission_status(submission_id)
+                submission_status = SubmissionWSClient(username, password).get_submission_status(submission_id)
```

Comment:
Passing the username and password is required because the `auth` is a global variable and subsequent call will not have the username/password if it is initially created without them.

Reply:
I did try briefly and it didn't work first time so didn't push it since it wasn't pat of this ticket